schema/config_pointset_point.json

```diff
@@ -5,7 +5,9 @@
   "additionalProperties": false,
   "properties": {
     "ref": {
-      "type": "string"
+      "type": "string",
+      "description": "Local reference for the point used for mapping data points to their respective UDMI datapoint name. For example, it could be the internal point name or a BACnet object reference",
```

Comment:
difficult to read -- maybe just "Mapping for the point to an internal resource (e.g. BACnet object reference)"?